thewtex/itk-wasm
# umd example update

- ci(UMDExample): Increase timeout
- build(UMDExample): Bump Cypress to 9.5.2
- feat(itk-wasm-cli): Update default Docker image for 1.0.0-b.6
- feat(version): Bump version to 1.0.0-b.6
- doc(runPipeline): Migrate runPipelineBrowser -> runPipeline
- build(Dependencies): Bump typescript to 4.6.2
- doc(README): Add license, commit activity badges
- ci(WebpackExample): Increase Chrome, Electron timeout
- doc(DebuggingExample): Bump to itk-wasm 1.0.0-b.6
- build(RapidJSON): Avoid use CMake file RELATIVE_PATH with absolute path
- doc(HelloWorld): Bump to 1.0.0-b.6
- build(libcbor): Remove target_include_directories with build tree
- build(libcbor): Remove target_include_directories with source tree
- doc(Node.jsExample): Bump to itk-wasm 1.0.0-b.6
- build(exports): Export getFileExtension
- doc(UMDExample): Bump to 1.0.0-b.6
- doc(WebpackExample): Update to 1.0.0-b.6
- doc(Webpack): Remove unused karma.conf.js
- doc(WebPackExample): Describe itkConfig
- feat(ImportVectorImageFilter): Initial addition
- feat(WASMImageToImageFilter): VectorImage support
- feat(SupportInputImageTypesNoVectorImage): Initial addition
- build(WASMMeshToMeshFilter): Address shadowing numberOfPoints
- build(WASMMeshToMeshFilter): Comment unused variables
- build(NPMPackage): Exclude doc/
- test(MedianFilterTest): Use SupportInputImageTypesNoVectorImage
- fix(PipelineInputsOutputs): Skip IO on lexical_cast with empty string
- build(Emscripten): Improve build flags
- feat(itk-wasm-cli): Update default Docker image for 1.0.0-b.8
- feat(version): Bump version to 1.0.0-b.8
- build(extensionTo{Image,Mesh}IO): Export from package
- test(WebpackExample): Bump viewer version
- test(WebpackExample): Install cypress for Firefox in Docker execution
- build(deps): Bump minimist from 1.2.5 to 1.2.6 in /examples/HelloWorld
- build(deps): Bump minimist from 1.2.5 to 1.2.6 in /examples/UMD
- doc(UnpkgExample): Remove -- no longer needed
